ui/pages/confirmations/hooks/setConfirmationAlerts.ts

```diff
@@ -1,17 +1,19 @@
-import { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
 
 import {
   clearAlerts,
   updateAlerts,
 } from '../../../ducks/confirm-alerts/confirm-alerts';
 import { currentConfirmationSelector } from '../../../selectors';
-import useConfirmationAlerts from './useConfirmationAlerts';
+import useBlockaidAlerts from './alerts/useBlockaidAlert';
 
 const setConfirmationAlerts = () => {
   const dispatch = useDispatch();
+  // todo: currently we are showing only blockaid alerts
+  // more will be added to this list
+  const alerts = useBlockaidAlerts();
   const currentConfirmation = useSelector(currentConfirmationSelector);
-  const alerts = useConfirmationAlerts();
```

Comment:
The only thing `setConfirmationAlerts` has more than `useConfirmationAlerts` is updating the state. Thus 2 hooks here look bit redundant.

Reply:
For current confirmation we have 2 hooks `useCurrentConfirmation` and `setCurrentConfirmation` as `useCurrentConfirmation` has a use case outside `setCurrentConfirmation`.

Reply:
Updating the state is the central purpose of `setConfirmationAlerts` though.

My concern is that in future we could have many many hooks for each approval type for example, and we'd have to reference each one directly within this hook, but it would be cleaner and easier to maintain them in an aggregate hook like `useConfirmationAlerts`.

Reply:
I see, let me revert this change.